README.md

```diff
@@ -33,6 +33,7 @@ Example: `geordi branch`
 On the first execution we ask for your Pivotal Tracker API token. It will be
 stored in `~/.config/geordi/global.yml`.
 
+By default `geordi branch` will checkout a new branch based on the current branch, you can use `geordi branch -m` to checkout from master.
```

Comment:
Caution: These docs are auto-generated from command descriptions by `rake readme`. Since you have documented the option there, this line could IMO be omitted.